odoo-shippable/scripts/build-image.sh

```diff
@@ -235,6 +235,26 @@ git_ps1_style(){
 PS1=$UserMachine$Color_Off$PathShort\$\\n"\$(git_ps1_style)"$Color_Off\$" "
 EOF
 
+# Add alias for multitail and configure
+cat >> /etc/bash.bashrc << EOF
+alias tail2="multitail -cS log4j"
+EOF
+
+cat >> /etc/multitail.conf << EOF
+# Odoo log
+colorscheme:odoo
```

Comment:
If you are adding a new schema, then this one should be used from `multitail -cS odoo` or not?